release-notes.md

```diff
@@ -1,5 +1,17 @@
 # Release notes
 
+## Sep 9, 2019
+
+The following updates were made in this release:
+
+**Android Griffon SDK 1.0.0**
+
+* New Artifact split out from the Android Griffon Bridge as of version 1.0.3.
+
+**Android Griffon Bridge 1.0.3**
+
+* Use with new Android Griffon SDK 1.0.0 or newer.
```

Comment:
- Please reword to: 

"You can use the Android Griffon Bridge version 1.0.3 with the Android Griffon SDK version 1.0.0.

- You're mentioning the Griffon Bridge in the 1st bullet but only introducing it in the 2nd bullet. It might be better to switch the bullets around.